docs/README.md

```diff
@@ -326,11 +326,18 @@ The following options are supported by all filters except `Callback` and `Finder
 - `defaultValue` (`mixed`, defaults to `null`) The default value that is being
   used in case the value passed for the corresponding field is invalid or missing.
 
+- `beforeProcess` (`callable`, defaults to `null`) A callable which can be used
+  to modify the query before the main `process()` method of filter is run.
+  It receives `$query` and `$args` as arguments. You can use the callback for e.g.
```

Comment:
We might need an example of the structure of such a callback. An example on how to use it for default case should suffice (void/null return).

Reply:
Done in https://github.com/FriendsOfCake/search/commit/8731c04c637198ae0fb37e943f617229a8997931